Source/Tests/LinuxSyscalls/Syscalls.h

```diff
@@ -84,24 +84,6 @@ struct ExecveAtArgs {
 
 uint64_t ExecveHandler(const char *pathname, char* const* argv, char* const* envp, ExecveAtArgs *Args);
 
-using RegisterSyscallInternalType =
-void (*)(int SyscallNumber,
-  int32_t HostSyscallNumber,
-  FEXCore::IR::SyscallFlags Flags,
-#ifdef DEBUG_STRACE
-  const std::string& TraceFormatString,
-#endif
-  void* SyscallHandler, int ArgumentCount);
-
-
-void RegisterSyscallInternalNop(int SyscallNumber,
```

Comment:
Is this still used somewhere? If not, then the definition in Syscalls.cpp should also be removed.

Reply:
Oops, good catch. Yep, removed.